projects/ADuCM360_demo_adxl355_pmdz/src/main.c

```diff
@@ -132,7 +132,7 @@ int main(int argc, char *argv[])
 
 {
    float volatile f32temp = 0.0f;
-   float adxl355Scale;
+   float adxl355Scale 0.0f;
```

Comment:
isn't this missing a `=` sign ?

Reply:
My bad. Just fixed it in the current version.